packages/client-core/src/components/Glass/ChatMenu.tsx

```diff
@@ -80,6 +85,25 @@ const OtherChat = ({ children }) => (
 )
 
 export const ChatMenu = () => {
+  const { type } = useMutableState(AuthState).authUser.identityProvider.get()
+  const onCTAClicked = () => {
+    ModalState.openModal(<SettingsMenu initScreen="signup" onClose={ModalState.closeModal} />)
+  }
+  if (type === 'guest') {
+    return (
+      <div className="flex h-full max-w-screen-sm flex-col items-center justify-center gap-8 font-dm-sans">
+        <HiChatBubbleLeftRight className="mx-auto h-[5.5rem] w-[5.5rem]" />
+        <div className="text-shadow-md text-2xl text-white">Want to chat with others?</div>
+        <button
```

Comment:
just a note, are we updating our existing UI library with the new designs? if not, we probably should, such that we can re-use things rather than having all sorts of buttons, sliders etc hardcoded all over the place.